tools/src/Logger.ts

```diff
@@ -0,0 +1,33 @@
+export enum LogLevel {
+  info = 1,
+  warn = 2,
+  error = 3
```

Comment:
In all the loggers the order is usually the other way around, you should flip these.

https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels

Reply:
Ahhh Today I learned. Thanks

Reply:
It's been updated now. I've squashed all commits too. Feel free to merge it after approving.